data/common.yaml

```diff
@@ -117,6 +119,26 @@ scriptworker_users:
         - notary8
         - notary9
         - notary10
+  vpn:
+    cltbld:
+      home: /builds/scriptworker
+      supported_behaviors:
+        - mac_notarize_vpn
+      notarization_users:  # Do we need this?
+        - notary1
+      dmg_prefix: "prod"
+      cot_product: "mozillavpn"
+      keychain_filename: "mozilla.20220330.keychain"
+      ed_key_filename: "ed25519_privkey"
+    # TODO: Split dep into it's own worker
+    depbld1:
```

Comment:
Unfortunately having dep on the same worker will require extensive changes to puppet logic. As is, the current setup doesn't support having 2 users (dep/prod) with different secrets.
ie: cltbld{1,2,3} will all have the same secrets, cannot have cltbld and dep under the same role.

@escapewindow we could move vpn dep to the dep workers along with ff and tb. Thoughts?

Reply:
Can do, I think. It might be nice to have memory/cpu nagios alerts set up so we know if the dep boxes are getting overloaded.